client/js/render.js

```diff
@@ -101,6 +102,10 @@ function buildChatMessage(msg) {
 		renderPreview(preview, renderedMessage);
 	});
 
+	if (template === "msg" && type !== "motd" && (msg.from === "" || type === "error")) {
```

Comment:
Can you just send `translate: true` from the server where it sends tokens instead?

Reply:
Done